src/main/java/com/faforever/api/error/ErrorCode.java

```diff
@@ -52,6 +52,13 @@
   MAP_FILE_INSIDE_ZIP_MISSING(145, "File is missing", "Cannot find inside the zip file {0}"),
   MAP_NO_VALID_JSON_METADATA(146, "No valid json", "metadata json is not valid"),
   MAP_RENAME_FAILED(147, "Cannot rename to correct name failed ", "Cannot rename file {0}"),
+  CLAN_CREATE_CREATOR_IS_IN_A_CLAN(201, "You are already in a clan", "Clan creator is already member of a clan"),
+  CLAN_ACCEPT_TOKEN_EXPIRE(203, "Token Expire", "The Invitation Link expire"),
+  CLAN_ACCEPT_WRONG_PLAYER(204, "Wrong Player", "Your are not the invited player"),
+  CLAN_ACCEPT_PLAYER_IN_A_CLAN(205, "Player is in a clan", "You are allready in a clan"),
+  CLAN_NOT_LEADER(206, "You Permission", "You are not the leader of the clan"),
+  CLAN_NOT_EXISTS(207, "Cannot find Clan", "Clan with id {0} is not available"),
+  CLAN_GENERATE_LINK_PLAYER_NOT_FOUND(208, "Player not found", "Cannot find player with id {0} who should be invited to the clan"),
```

Comment:
Always quote placeholders like this: `'{0}'`

Why: Once you wasted time trying to figure out why you're getting an `Invalid attribute name` exception and what the freaking attribute name is, just to figure out that `name` is the attribute, you'll know why:
`Invalid attribute 'name'` or `Invalid attribute: name` oooh ok easy!